src/mopidy/core/playback.py

```diff
@@ -339,8 +327,6 @@ def play(  # noqa: C901, PLR0912
                 logger.info("No playable track in the list.")
                 break
 
-        # TODO return result?
```

Comment:
Sure? It's still.not a bad idea. Especially if we did decide to return early when the tlid isn't found (my earlier comment)

Reply:
I was just pruning ancient comments. I think the comment being present or not wouldn't increase or decrease the chances of us adding success return values to more of our APIs.